htdocs/comm/action/class/actioncomm.class.php

```diff
@@ -1201,13 +1201,20 @@ public function load_board($user, $load_state_board = 0)
     	}
     	$sql .= " FROM ".MAIN_DB_PREFIX."actioncomm as a";
     	if (!$user->rights->societe->client->voir && !$user->socid) $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."societe_commerciaux as sc ON a.fk_soc = sc.fk_soc";
+    	if (!$user->rights->agenda->allactions->read) { // If bloack Added by PV
+    	    $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."actioncomm_resources AS ar ON a.id = ar.fk_actioncomm";
```

Comment:
If the event is assigned to n record in resources, adding this will duplicate the line by n so will make the count wrong.
The left join here should be:
$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."actioncomm_resources AS ar ON a.id = ar.fk_actioncomm AND  ar.element_type ='user' AND ar.fk_element = ".$user->id";
So only 1 record can be found into child table. Can you check this ?

Reply:
Thanks for checking @eldy , will try your suggestion. Will revert back asap